run_tests.lua

```diff
@@ -1,3 +1,7 @@
+-- The test script to run busted tests in a separate headless nvim process
+
+local busted = require("plenary.busted")
```

Comment:
Not needed

Reply:
I thought importing busted as soon as possible would be a better idea because it gives more intuitive stacktrace, but since it's not related to this PR I can fix as suggested.